plugins/spark/pyspark_examples/README.md

```diff
@@ -0,0 +1,23 @@
+# Simple PySpark Job as a Flyte Workflow
+
+[Example: pyspark_pi.py](pyspark_pi.py)
+
+## Introduction:
+
+In this example, we run a simple PySpark job as part of a Flyte workflow. The Spark Job calculates the value of Pi and sets it as the output of the task.
```

Comment:
Example that shows how two different tasks (spark and python) can be built using the same docker image

Reply:
done